core/templates/dev/head/components/state-editor/state-responses-editor/state-responses.directive.ts

```diff
@@ -236,6 +252,17 @@ oppia.directive('stateResponses', [
             return interactionId && INTERACTION_SPECS[interactionId].is_linear;
           };
 
+          $scope.isCurrentInteractionTrivial = function() {
+            var interactionId = $scope.getCurrentInteractionId();
+            if (
+              constants.INTERACTION_IDS_WITHOUT_ANSWER_DETAILS.indexOf(
```

Comment:
You can use ` return INTERACTION_IDS_WITHOUT_ANSWER_DETAILS.includes(interaction_id)`. (I'm fine with the current implementation as well.)

Reply:
Okay

Reply:
Done